lib/irb/cmd/subirb.rb

```diff
@@ -29,6 +28,12 @@ class IrbCommand < MultiIRBCommand
       description "Start a child IRB."
 
       def execute(*obj)
+        if irb_context.with_debugger
```

Comment:
Is it possible to put this in the parent class and invoke super to avoid duplicating the message? Or does that not make sense?

Reply:
Updated 👍